google/resumable_media/requests/__init__.py

```diff
@@ -665,7 +665,15 @@ def SimpleUpload(*args, **kwargs):
     pkg_resources.require('requests >= 2.18.0')
 except pkg_resources.ResolutionError as caught_exc:  # pragma: NO COVER
     import six
+    message = ""
+    if isinstance(caught_exc, pkg_resources.VersionConflict):
+        message = "There is a version conflict in request module '
+        'required by ``google.resumable_media.requests`` subpackage.\n'
```

Comment:
In order to span multiple lines, you need either, a slash or triple quotes.
Also, there is a typo: a double quote is used in the beginning while a single quote - at the end of the string.

Reply:
My bad, slash is required at end of string. Now added. Also removed double quote.

Reply:
Do we require 'message' variable declaration, since it is getting set in if else statement anyways.
Good to have though, can you make it with single quote just to maintain the consistency.
message = ''

Reply:
We need message declaration for the case when we dont have both the exceptions. In that case, if else statement wont hit.
Yes, i will make message = '' (single quote)

Reply:
Looks good. Plz open public PR